zone/spells.cpp

```diff
@@ -3161,6 +3161,12 @@ int Mob::CheckStackConflict(uint16 spellid1, int caster_level1, uint16 spellid2,
 		}
 		if (sp2_value != sp1_value)
 			values_equal = false;
+
+		if (RuleB(Spells, ResurrectionEffectsBlock) && IsResurrectionEffects(spellid1)) {
```

Comment:
Haven't gotten a clear answer from anyone. Is this live behavior? If so, we could remove the rule entirely and leave this as sane default

Reply:
I cannot confirm whether or not this is true live behavior, however it does exist on p99 which is what I based the rule off of. (Not that it means it's correct or not, but it's the closest thing I can go off of)

Reply:
https://github.com/EQEmu/Server/pull/2990#issuecomment-1454754358